tests/integration/helpers.bash

```diff
@@ -19,6 +19,9 @@ HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 # CRIU PATH
 CRIU="/usr/local/sbin/criu"
 
+# CONFIG_MEMCG_KMEM support
+KMEM="/sys/fs/cgroup/memory/memory.kmem.limit_in_bytes"
```

Comment:
@cyphar just pointed out that you might be implicating that cgroups might not be mounted on `/sys/fs/cgroup` but somewhere else and your idea is to parse `/proc/self/mountinfo` to find the cgroup mountpoint.